cli/src/logger/mod.rs

```diff
@@ -15,9 +15,7 @@ pub fn init() -> Result<(), SetLoggerError> {
   let sematext = Box::new(Sematext);
   let logzio = Box::new(LogzIO);
   match env.as_str() {
-    "local" => {
-      multi_log::MultiLogger::init(vec![simple_logger, sematext, logzio], log::Level::Info)
-    }
-    _ => multi_log::MultiLogger::init(vec![sematext], log::Level::Info),
+    "local" => multi_log::MultiLogger::init(vec![simple_logger], log::Level::Info),
+    _ => multi_log::MultiLogger::init(vec![sematext, logzio], log::Level::Info),
```

Comment:
Same here. Are we going to use two logging backends, or will we select one before this is merged?

Reply:
Will keep only one after trial period.